### PR TITLE
REP-4524 Additional Header population and doc migration

### PR DESCRIPTION
**Pull request by adrianjgeorge**

### CODE REVIEW

File: repose-aggregator/src/docs/asciidoc/filters/index.adoc [hunk 1]
```diff
@@ -8,6 +8,9 @@ include::../_includes/in-progress.adoc[]
 <<cors.adoc#,CORS Filter>>::
     Enables users to add support for Cross-Origin Resource Sharing requests.
 
+<<rackspace-auth-user.adoc#,Rackspace Auth User>>::
```
Comment (Mario-Lopez) [inline repose-aggregator/src/docs/asciidoc/filters/index.adoc]: Missing "Filter" at the end of the name for consistency.

**Comment by Mario-Lopez** [comment]
The filter unit test looks pretty weak, and then I saw my name is all over it.  Apparently I added the unit test when we updated the wrappers and didn't add tests for the other stuff the filter was doing.  I guess it's a good thing we're about to refactor the whole thing.

**Comment by adrianjgeorge** [comment]
Yeah, it was pretty close to non-existent.

File: repose-aggregator/src/docs/asciidoc/filters/index.adoc (hunk 1 above)
Comment (adrianjgeorge) [inline repose-aggregator/src/docs/asciidoc/filters/index.adoc]: Fixed

**Pull request merged**